app/src/main/java/io/pslab/activity/PowerSourceActivity.java

```diff
@@ -103,7 +116,7 @@
     ImageView bottomSheetSchematic;
     @BindView(R.id.custom_dialog_desc)
     TextView bottomSheetDesc;
-
+    private PowerSourceData powerSourceData = new PowerSourceData();
```

Comment:
@ho-dor Why are you creating the object directly here, shouldn't it be more appropriate if it is declared in onCreate method